src/goInstallTools.ts

```diff
@@ -443,6 +453,28 @@ export function offerToInstallTools() {
 				});
 			}
 		});
+
+		const usingSourceGraph = getToolFromToolPath(getLanguageServerToolPath()) === 'go-langserver';
+		if (usingSourceGraph) {
+			const promptMsg = 'The language server from Sourcegraph is no longer under active development and it does not support Go modules as well. Please install and use the language server from Google or disable the use of language servers altogether.';
+			vscode.window.showInformationMessage(promptMsg, 'Install', 'Disable language server')
```

Comment:
>Install', 'Disable language server [](start = 52, length = 34)

can we extract these as const and reuse them in the check and here to reduce the chance of some misspelling or going out of sync by mistake

Reply:
Done